smart/plugin.go

```diff
@@ -166,6 +177,10 @@ func (sc *SmartCollector) GetMetricTypes(_ plugin.ConfigType) ([]plugin.MetricTy
 
 //GetConfigPolicy returns a ConfigPolicy
 func (p *SmartCollector) GetConfigPolicy() (*cpolicy.ConfigPolicy, error) {
-	c := cpolicy.New()
-	return c, nil
+	cp := cpolicy.New()
+	rule, _ := cpolicy.NewStringRule("proc_path", false, "/proc")
+	node := cpolicy.NewPolicyNode()
+	node.Add(rule)
+	cp.Add([]string{nsVendor, nsClass, PluginName}, node)
+	return cp, nil
```

Comment:
In some cases it might be easier to user environment variable for passing configuration to plugin.
Please consider reading environment variable as a primary definition for the /procfs mount point.